### PR TITLE
Branch rene

### PR DESCRIPTION
**Pull request by renechoy66**
- Level 1 domain with discretization files
- vi agent
- agent template for other algos

### CODE REVIEW

File: MarsRoverDisc.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+from typing import Optional, Tuple
```
Comment (lowkangn) [inline MarsRoverDisc.py]: What is this import from actually?
If it's not used, could you help to remove it?

File: README.md [hunk 2]
```diff
@@ -1 +1,6 @@
-# mars-rover
\ No newline at end of file
+# mars-rover
+
+## level 1
+Changes: 
+- point minerals: AREA attribute has deault of 0.5. remove references to AREA in instance files
```
Comment (lowkangn) [inline README.md:5]:
```suggestion
- Point minerals: AREA attribute has default of 0.5. remove references to AREA in instance files.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,6 @@
...
+- moving mechanism based on x-y displacement only. removed all references to power, velocity and u
```
Comment (lowkangn) [inline README.md:6]:
```suggestion
- Moving mechanism based on x-y displacement only. Removed all references to power, velocity and u.
```

File: agent_template.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		print('action     = {}'.format(action), myEnv.disc_actions[action])
+		print('next state = {}'.format(next_state), myEnv.disc_states[next_state])
+		print('reward     = {}'.format(reward))
+		state = next_state
+		if done:
+			break
+	print("episode ended with reward {}".format(total_reward))
+	myEnv.close()
+
+main()
```
Comment (lowkangn) [inline agent_template.py:60]: Missing new line. Actually quite a few files are missing it but it's just a minor thing.

**Review by lowkangn — APPROVED** [review]
LGTM, just a few nagging issues to iron out and it's good to merge 👍

File: MarsRoverDisc.py (hunk 1 above)
Comment (renechoy66) [reply MarsRoverDisc.py]: dk from where one, deleted

**Pull request merged**